pkg/alertmanager/types.go

```diff
@@ -354,8 +355,8 @@ type discordConfig struct {
 type webexConfig struct {
 	VSendResolved *bool             `yaml:"send_resolved,omitempty"`
 	HTTPConfig    *httpClientConfig `yaml:"http_config,omitempty"`
-	APIURL        string            `yaml:"api_url,omitempty"`
-	Message       string            `yaml:"message,omitempty"`
+	APIURL        *string           `yaml:"api_url,omitempty"`
+	Message       *string           `yaml:"message,omitempty"`
```

Comment:
I wouldn't change the types here as they come from the Alertmanager upstream configuration format.